rtl-spec/commands-address-bar.spec.tsx

```diff
@@ -62,10 +63,17 @@ describe('AddressBar component', () => {
     GistActionState.publishing,
     GistActionState.updating,
   ])('disables during active gist action (%s)', (action) => {
-    const { getByRole } = render(
+    const gistId = '159cb99a70a201bd5e08194674f4c571';
+    const gistUrl = `https://gist.github.com/ghost/${gistId}`;
+    const { getByPlaceholderText, getByRole } = render(
       <AddressBar appState={(store as unknown) as AppState} />,
     );
-    store.activeGistAction = action;
+    fireEvent.change(getByPlaceholderText('https://gist.github.com/...'), {
+      target: { value: gistUrl },
+    });
```

Comment:
Maybe we should wait for https://github.com/electron/fiddle/pull/1099 to merge first. Because of this, we can avoid using the fireEvent method and instead use the `type` API in `@testing-library/user-event`

e.g: 
```ts
userEvent.type(getByPlaceholderText('https://gist.github.com/...'), gistUrl)
```

https://testing-library.com/docs/ecosystem-user-event#typeelement-text-options

Reply:
Update: #1099 already merged.